_generated/NuGetAuthenticateV1/taskJsonOverride.loc.json

```diff
@@ -10,15 +10,18 @@
             "properties": {
               "EditableOptions": "False",
               "MultiSelectFlatList": "False"
-            }
+            },
+            "visibleRule": "nuGetServiceConnections == ''"
         },
         {
             "name": "feedUrl",
+            "aliases": ["adoServiceConnectionCrossOrgFeedUrl"],
```

Comment:
the other parameter is azureDevOpsServiceConnection but this new one uses the acronym ADO, why the difference here? since these are custom visible I would recommend we avoid acronyms, abbreviations, etc. that might not be clear.

Reply:
Agreed. Looking at [tasks.schema.json](https://github.com/microsoft/azure-pipelines-task-lib/blob/master/tasks.schema.json), I don't see a limitation imposed on the parameter name. I tested the longer `azureDevOpsServiceConnectionCrossOrgFeedUrl` and ran into no issues with it. I'm sticking with this, since it gets us close to our goal of parity with the other input, and accuracy on it's purpose.